.github/pull_request_template.md

```diff
@@ -1,3 +1,8 @@
+<!--
+  How to contribute: https://vitess.io/docs/contributing/
+  We’re looking forward to any contribution! Before you start larger contributions, make sure to reach out first and discuss your plans with us.
```

Comment:
`us` should link somewhere useful. (I know the doc on the website has the same language and the same problem).
How about linking it to the #developers slack channel?

Reply:
Agreed that "us" is vague. This line was copied from https://vitess.io/docs/contributing/ by way of TL;DR. I'm not sure we want to ask people to discuss all contributions on #developers . We should rather encourage them to open an issue. What do you think?

Reply:
Yes! Let us ask them to create an issue to document the feature, and if they have a proposal it can be marked as an RFC.

Reply:
Updated.